sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -283,13 +289,18 @@ export class ConfirmationOfEnrollmentController extends BaseController {
    * @param locationId location id of the application
    * @param disbursementScheduleId disbursement schedule id of COE
    */
+  @ApiUnprocessableEntityResponse({
+    description:
+      "Tuition amount provided should be lesser than both (Actual tuition + Program related costs) and (Canada grants + Canada Loan + BC Loan). OR First disbursement(COE) not complete. Please complete the first disbursement.",
```

Comment:
Please convert to a regular sentence (`Loan + BC Loan). OR First disbursement(COE)`???).